stable/artifactory-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: artifactory-ha
 home: https://www.jfrog.com/artifactory/
-version: 0.1.3
+version: 0.1.5
```

Comment:
Not entirely sure why you bumped to 0.1.5, when 0.1.4 is not in use yet. That said, this change warrants a bump of the minor version, as you're adding new features.

Reply:
fixed